csun-metalab/nomi
# Nam 86 -- Quick And Dirty Card Shuffling

A simple solution to passing student information to the front end as well as shuffling student data.

Testing:
1. Update .env, I emailed the values for security purposes
2. Log in as matthew.fritz
3. make sure every link in the home page works
